test/src/test/java/org/corfudb/runtime/view/ObjectsViewTest.java

```diff
@@ -39,20 +39,21 @@ public void canAbortNoTransaction()
     public void abortedTransactionDoesNotConflict()
             throws Exception {
         final String mapA = "map a";
-        //Enable transaction logging
-        CorfuRuntime r = getDefaultRuntime()
-                .setTransactionLogging(true);
+        final String streamA = "streamA";
+        CorfuRuntime r = getDefaultRuntime();
 
         CorfuTable<String, String> map = getDefaultRuntime().getObjectsView()
                 .build()
                 .setStreamName(mapA)
+                .setStreamTags(CorfuRuntime.getStreamID(streamA))
```

Comment:
I don't understand what is being done here, can you explain?

Reply:
This test verifies that if a tx aborted, its updates will not be present in the transaction stream.  Since we have removed the transaction stream, I am not using the tag "streamA" for the table(mapA).  

It is also possible to test without using the tags and only polling on the stream id for mapA, but it will require more changes to the test.  Let me know if you would like to get more background and discuss further.

Reply:
Ok, yeah sounds reasonable to keep testing this on the stream tags. Looks good!